packages/blade/src/components/DatePicker/__tests__/DatePicker.test.stories.tsx

```diff
@@ -22,7 +22,7 @@ export const DatePickerShouldShow: StoryFn<typeof DatePickerComponent> = (): Rea
 
 DatePickerShouldShow.play = async () => {
   const { getByRole, queryByText } = within(document.body);
-  const input = getByRole('combobox', { name: /Select Date/i });
+  const input = getByRole('textbox', { name: /Select Date/i });
```

Comment:
Technically role can still be "combobox" since it has a dropdown (ie datepicker overlay) 

<img width="782" height="114" alt="image" src="https://github.com/user-attachments/assets/2636f73f-cabf-4a44-8a9d-157808573ed7" />